native/example_low_light_image_enhancement/nnstreamer_example_low_light_image_enhancement.py

```diff
@@ -0,0 +1,145 @@
+        self.pipeline = Gst.parse_launch(
+            'filesrc location=' + self.image_name + ' ! pngdec ! videoscale ! '
+            'videoconvert ! video/x-raw,width=600,height=400,format=RGB ! tensor_converter ! '
+            'tensor_transform mode=arithmetic option=typecast:float32,add:0,div:255.0 ! '
+            'tensor_filter framework=tensorflow-lite model= '+ self.tflite_model +' ! '
+            'tensor_sink name=tensor_sink'
+        )
+
+        # bus and message callback
+        bus = self.pipeline.get_bus()
+        bus.add_signal_watch()
+        bus.connect('message', self.on_bus_message)
+
+        # tensor sink signal : new data callback
+        tensor_sink = self.pipeline.get_by_name('tensor_sink')
+        tensor_sink.connect('new-data', self.on_new_data)
+
+        # start pipeline
+        self.pipeline.set_state(Gst.State.PLAYING)
+        self.running = True
+
+        # run main loop
+        self.loop.run()
+
+        # quit when received eos or error message
+        self.running = False
+        self.pipeline.set_state(Gst.State.NULL)
+
+        bus.remove_signal_watch()
+
+    def on_bus_message(self, bus, message):
+        """Callback for message.
+        :param bus: pipeline bus
+        :param message: message from pipeline
+        :return: None
+        """
+        if message.type == Gst.MessageType.EOS:
+            logging.info('received eos message')
+            self.loop.quit()
+        elif message.type == Gst.MessageType.ERROR:
+            error, debug = message.parse_error()
+            logging.warning('[error] %s : %s', error.message, debug)
+            self.loop.quit()
+        elif message.type == Gst.MessageType.WARNING:
+            error, debug = message.parse_warning()
+            logging.warning('[warning] %s : %s', error.message, debug)
+        elif message.type == Gst.MessageType.STREAM_START:
+            logging.info('received start message')
+        elif message.type == Gst.MessageType.QOS:
+            data_format, processed, dropped = message.parse_qos_stats()
+            format_str = Gst.Format.get_name(data_format)
+            logging.debug('[qos] format[%s] processed[%d] dropped[%d]', format_str, processed, dropped)
+
+    def on_new_data(self, sink, buffer):
+        """Callback for tensor sink signal.
+        :param sink: tensor sink element
+        :param buffer: buffer from element
+        :return: None
+        """
+        if self.running:
+            for idx in range(buffer.n_memory()):
+                mem = buffer.get_memory(idx)
+                result, mapinfo = mem.map(Gst.MapFlags.READ)
+
+                if result:
+                    content_arr = np.frombuffer(mapinfo.data, dtype=np.float32)
+                    content = np.reshape(content_arr,(400,600,3)) * 255.0
+                    content = content.clip(0,255)
+                    img = Image.fromarray(np.uint8(content))
+                    img.save('low_light_enhancement_'+self.image_name)
```

Comment:
How about showing the result image so that the user can check the result right away?
```suggestion
                    img.save('low_light_enhancement_'+self.image_name)
                    img.show()
```